<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LASSIE v1.0.0 </string>
  </property>
  <property name="windowIcon">
   <iconset resource="lassie-resources.qrc">
    <normaloff>:/logo/images/lassie-icon.png</normaloff>:/logo/images/lassie-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="openmodelbutton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Open &amp;model</string>
        </property>
        <property name="icon">
         <iconset resource="lassie-resources.qrc">
          <normaloff>:/logo/images/open-directory.png</normaloff>:/logo/images/open-directory.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="input_greenlight">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="lassie-resources.qrc">:/logo/images/check.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="inputpathlabel">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:gray</string>
        </property>
        <property name="text">
         <string>Please select an input directory</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Model's summary</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTableWidget" name="speciestable">
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>40</number>
             </attribute>
             <column>
              <property name="text">
               <string>Species name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Initial amount</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Plot</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="selallbutton">
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Select all species</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deselallbutton">
             <property name="font">
              <font>
               <pointsize>7</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Deselect all species</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QTableWidget" name="reactionstable">
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <column>
            <property name="text">
             <string>Reaction</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Kinetic parameter</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="outputdirectorybutton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&amp;Choose output directory</string>
        </property>
        <property name="icon">
         <iconset resource="lassie-resources.qrc">
          <normaloff>:/logo/images/store-directory.png</normaloff>:/logo/images/store-directory.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="output_greenlight">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="lassie-resources.qrc">:/logo/images/check.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="outputpathlabel">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:gray</string>
        </property>
        <property name="text">
         <string>Please select an output directory</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Simulation time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="timemax">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Samples:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="samples">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QCommandLinkButton" name="simulatebutton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Simulate model</string>
        </property>
        <property name="icon">
         <iconset resource="lassie-resources.qrc">
          <normaloff>:/logo/images/function.png</normaloff>:/logo/images/function.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="save_to_file">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save output dynamics</string>
        </property>
        <property name="icon">
         <iconset resource="lassie-resources.qrc">
          <normaloff>:/logo/images/download-icon.png</normaloff>:/logo/images/download-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="action_Specify_LASSIE_s_executable_file"/>
    <addaction name="actionSpecify_output_directory"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open_model"/>
    <addaction name="action_Import_SBML_experimental"/>
    <addaction name="separator"/>
    <addaction name="actionSave_dynamics"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_LASSIE"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QDockWidget" name="outputwidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>79</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Output dynamics</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="plotlayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Specify_LASSIE_s_executable_file">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Specify LASSIE's executable file...</string>
   </property>
  </action>
  <action name="action_Open_model">
   <property name="text">
    <string>&amp;Open model...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSpecify_output_directory">
   <property name="text">
    <string>Choose output &amp;directory...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Import_SBML_experimental">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Import SBML...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionAbout_LASSIE">
   <property name="text">
    <string>About LASSIE</string>
   </property>
  </action>
  <action name="actionAbout_this_GUI">
   <property name="text">
    <string>About this GUI</string>
   </property>
  </action>
  <action name="actionSave_dynamics">
   <property name="text">
    <string>Save &amp;dynamics...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="lassie-resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openmodelbutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_model()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputdirectorybutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>choose_output_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simulatebutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>simulate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>623</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Open_model</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_model()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSpecify_output_directory</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>choose_output_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Import_SBML_experimental</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>import_SBML()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selallbutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>select_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deselallbutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deselect_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_LASSIE</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Specify_LASSIE_s_executable_file</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_options()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_to_file</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>save_files()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_model()</slot>
  <slot>choose_output_dir()</slot>
  <slot>simulate()</slot>
  <slot>import_SBML()</slot>
  <slot>select_all()</slot>
  <slot>deselect_all()</slot>
  <slot>show_help()</slot>
  <slot>show_options()</slot>
  <slot>save_files()</slot>
 </slots>
</ui>
